Makefile

```diff
@@ -18,29 +18,33 @@ run_pgstac = docker-compose run --rm \
 image:
 	docker-compose build
 
-.PHONY: docker-run
-docker-run: image
-	$(run_docker)
+.PHONY: docker-run-all
```

Comment:
added this so all the commands can be done with make rather than docker-compose.